src/OrleansAWSUtils/Streams/SQSStreamProviderUtils.cs

```diff
@@ -8,6 +8,13 @@ namespace OrleansAWSUtils.Streams
 {
     public class SQSStreamProviderUtils
     {
+        /// <summary>
+        /// Async method to delete all used queques, for specific provider and deploymentId
+        /// </summary>
+        /// <param name="providerName"></param>
+        /// <param name="deploymentId"></param>
+        /// <param name="storageConnectionString"></param>
+        /// <returns> Task object for this async method </returns>
```

Comment:
will do. yes I have did the same PR as your previous merged one and didn't realize it until I rebase master :<